aksel.nav.no/website/next-env.d.ts

```diff
@@ -2,4 +2,4 @@
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
-// see https://nextjs.org/docs/basic-features/typescript for more information.
+// see https://nextjs.org/docs/pages/building-your-application/configuring/typescript for more information.
```

Comment:
This change was made automatically by Nextjs. Maybe this file is not supposed to be committed? 🤔

Reply:
When you create a new nextjs project, this file is in .gitignore. Should we add it to our .gitignore in the website directory? @KenAJoh

Reply:
Makes sense to ignore it then if they want it ignored 👍 Can update "app-dir" with the ignore also then (maybe just ignore it on root? 🤔 )